MAINTAINERS.md

```diff
@@ -14,3 +17,15 @@
 
 ### Updating Practices
 To ensure common practices as maintainers, all practices are expected to be documented here or enforced through github actions.  There should be no expectations beyond what is documented in the repo [CONTRIBUTING.md](./CONTRIBUTING.md) and OpenSearch-Project [CONTRIBUTING.md](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md).  To modify an existing processes or create a new one, make a pull request on this MAINTAINERS.md for review and merge it after all maintainers approve of it.
+
+# Practices
+
+## Reverting Commits
+There will be changes that destabilize or block contributions.  The impact of these changes will be localized on the repository or even the entire OpenSearch project.  We should bias towards keeping contributions unblocked by immediately reverting impacting changes, these reverts will be done by a maintainer.  After the change has been reverted, an issue will be openned to re-merge the change and callout the elements of the contribution that need extra examination such as additional tests or even pull request workflows.
+
+Exceptional, instead of immediately reverting, if a contributor knows how and will resolve the issue in an hour we should fix-forward to reduce overhead.
+
+### Performing Revert
+Go to the pull request of the change that was an issue, there is a `Revert` button at the bottom.  If there are no conflicts to resolve, this can be done immediately bypassing standard approval.
+
+Reverts can also be done via the command line using `git revert` and creating a new pull request.  If done in this way they should have references to the pull request that was reverted.
```

Comment:
```suggestion
Reverts can also be done via the command line using `git revert <commit_id>` and creating a new pull request.  If done in this 
```

Reply:
Done in [1ba40dd](https://github.com/opensearch-project/security/pull/1754/commits/1ba40dd5e7aa2a8516e65e207f44f0501cbc536e)